src/main/java/edu/usf/cutr/trackerlib/utils/TimeUtils.java

```diff
@@ -25,7 +25,7 @@ public static long createBatchUpdateTimeMillis() {
         //Add two hours
         long twoHourMillis = 2*60*60*1000;
         // Test data
-        twoHourMillis = (60 * 1000);
+//        twoHourMillis = (60 * 1000);
```

Comment:
Yes, this was for test purposes (schedules batch updates every minute).  Maybe, I should put this line under ```if (BuildConfig.DEBUG) { ... } ``` .